10.0.0 Help Center Updates

vicenter-gitkraken opened the pull request:
Renames Launchpad, Code Suggestions, and updates Jira Data Center.

vicenter-gitkraken committed:
Rename Focus View to Launchpad

vicenter-gitkraken committed:
Changed screenshots and GIFs for new Launchpad

vicenter-gitkraken committed:
Focus View to Launchpad in Security page

jkelroy commented:
Hey @vicenter-gitkraken, while reviewing your PR, I'd suggest the following code changes:

👉 [**New Suggestions for you**](https://staging.gitkraken.dev/drafts/517433a2-da3c-4913-9b85-ee6f0dfcbfae?source=PRComment)
>

You can also review and apply these suggestions [locally on your machine](https://staging.gitkraken.dev/link/drafts/517433a2-da3c-4913-9b85-ee6f0dfcbfae).

<details><summary>Learn more about GitKraken Code Suggest</summary>

>[Code Suggest](https://www.gitkraken.com/solutions/code-suggest) liberates your code reviews from GitHub's restrictive, comment-only feedback style. As simple as suggesting changes in a Google-doc, provide real code suggestions from where you code, e.g. your IDE, and on anything in your project — not just on the lines of code changed in the PR.

_Join your team on [GitKraken](https://gitkraken.com) to speed up PR review._
</details>

jkelroy commented:
Hey @vicenter-gitkraken, while reviewing your PR, I'd suggest the following code changes:

👉 [**New Suggestions for you**](https://staging.gitkraken.dev/drafts/c738e9ca-25ab-4163-80cc-5f58b0f3b2c5?source=PRComment)
>

You can also review and apply these suggestions [locally on your machine](https://staging.gitkraken.dev/link/drafts/c738e9ca-25ab-4163-80cc-5f58b0f3b2c5).

<details><summary>Learn more about GitKraken Code Suggest</summary>

>[Code Suggest](https://www.gitkraken.com/solutions/code-suggest) liberates your code reviews from GitHub's restrictive, comment-only feedback style. As simple as suggesting changes in a Google-doc, provide real code suggestions from where you code, e.g. your IDE, and on anything in your project — not just on the lines of code changed in the PR.

_Join your team on [GitKraken](https://gitkraken.com) to speed up PR review._
</details>

jkelroy commented:
Hey @vicenter-gitkraken, while reviewing your PR, I'd suggest the following code changes:

👉 [**Code Suggestion for #215**](https://staging.gitkraken.dev/drafts/65e3460d-1489-452c-834f-5f437e14da90?source=PRComment)
>https://github.com/gitkraken/gitkraken-client-docs/pull/215

You can also review and apply these suggestions [locally on your machine](https://staging.gitkraken.dev/link/drafts/65e3460d-1489-452c-834f-5f437e14da90).

<details><summary>Learn more about GitKraken Code Suggest</summary>

>[Code Suggest](https://www.gitkraken.com/solutions/code-suggest) liberates your code reviews from GitHub's restrictive, comment-only feedback style. As simple as suggesting changes in a Google-doc, provide real code suggestions from where you code, e.g. your IDE, and on anything in your project — not just on the lines of code changed in the PR.

_Join your team on [GitKraken](https://gitkraken.com) to speed up PR review._
</details>

jkelroy commented:
Hey @vicenter-gitkraken, while reviewing your PR, I'd suggest the following code changes:

👉 [**Code Suggestion for #215**](https://staging.gitkraken.dev/drafts/2468909f-431b-49e1-ba0f-3c30a7d29963?source=PRComment)
>https://github.com/gitkraken/gitkraken-client-docs/pull/215

You can also review and apply these suggestions [locally on your machine](https://staging.gitkraken.dev/link/drafts/2468909f-431b-49e1-ba0f-3c30a7d29963).

<details><summary>Learn more about GitKraken Code Suggest</summary>

>[Code Suggest](https://www.gitkraken.com/solutions/code-suggest) liberates your code reviews from GitHub's restrictive, comment-only feedback style. As simple as suggesting changes in a Google-doc, provide real code suggestions from where you code, e.g. your IDE, and on anything in your project — not just on the lines of code changed in the PR.

_Join your team on [GitKraken](https://gitkraken.com) to speed up PR review._
</details>

vicenter-gitkraken committed:
Correct integration name for Jira Data Center and update references

- Corrected the integration name from "Jira Datacenter" to "Jira Data Center" for consistency.
- Updated references to the integration throughout the documentation.

vicenter-gitkraken committed:
Repo Management tab

vicenter-gitkraken committed:
Rewording

vicenter-gitkraken committed:
Open Repo modal disappears

vicenter-gitkraken committed:
Removed the vanishing act of repository deletion instructions

vicenter-gitkraken committed:
Favorites from Repo Management Tab

vicenter-gitkraken committed:
Renamed all Gitkraken Client occurrences to Gitkraken Destkop

vicenter-gitkraken committed:
Repo Management Tab screenshots

vicenter-gitkraken committed:
Code Suggestions in Gitkraken Desktop

vicenter-gitkraken committed:
Introduced dynamic GIF support for improved visual representation.

🚀 #CodeReviewEvolution

jkelroy commented on gitkraken-client/jira-data-center.md:
add redirect in wp

jkelroy commented on gitkraken-client/open-clone-init.md:
```diff
@@ -56,7 +71,7 @@ This will then prompt you to open the newly copied project in GitKraken Client.
 Starting a project in GitKraken Client is easy through <kbd><strong>File > Init</strong></kbd>
 
-<img src='/wp-content/uploads/init.png' srcset='/wp-content/uploads/init@2x.png 2x' class='img-bordered img-responsive center'>
+<img src='wp-content/uploads/gkc-repo-mngmt-init-repo.png' class='img-bordered img-responsive center'>
```
Do you know if images work without the leading backslash or should this (and a few others) be _src=`/wp-content/..._

jkelroy commented on gitkraken-client/pull-requests.md:
```diff
@@ -87,9 +92,19 @@ From this view, GitHub users may edit the pull request:
+
+### Review Code and Suggest Changes
+
+In Gitkraken Desktop, Review Code and Suggest Changes simplifies code review by allowing you to make suggestions and edits across the entire project, not just on the lines that were changed, GitKraken Desktop, and gitkraken.dev. When a Pull Request is open, you can make suggestions to the pull request that others can then review and accept to include in the pull request. 
```
> changed, GitKraken Desktop, and gitkraken.dev.

to 

> changed, in GitKraken Desktop, GitLens, and gitkraken.dev.

---

Also, we should include a section about the recipient opening and accepting the change

vicenter-gitkraken committed:
Release Notes for 10.0.0

vicenter-gitkraken committed:
We are back on the right timeline

vicenter-gitkraken committed:
Merge branch 'client-to-desktop' into 10-0-0-update
